Tools/sitl_run.sh

```diff
@@ -104,9 +104,9 @@ elif [ "$program" == "gazebo" ] && [ ! -n "$no_sim" ]; then
 				gzserver "$PX4_SITL_WORLD" &
 			fi
 		fi
+		SIM_PID=`echo $!`
```

Comment:
Don't overcomplicate this, it should just be `SIM_PID=$!`

Reply:
Right, follow up PR on https://github.com/PX4/Firmware/pull/15210